docs/tutorials/configure/pet-photographer.md

```diff
@@ -4,15 +4,11 @@ linkTitle: "Pet Photographer"
 type: "docs"
 description: "Use the filter modular component in the Viam app to photograph your pet in their collar."
 tags: ["vision", "filter", "camera", "detector", "services"]
-image: "/tutorials/pet-photographer/data-capture.png"
```

Comment:
This does unfortunately break previews. You can confirm here: https://docs-test.viam.dev/2457/components/base/ at the bottom. 

So when I said I wasn't sure whether you could remove one of them and the other would automatically use the same field I thought if you removed `images` it might automatically take the images available in `image`. It doesn't. Read [this here](https://gohugo.io/templates/internal/#configure-open-graph), it says "The first 6 URLs from the images array are used for image metadata. If [page bundles](https://gohugo.io/content-management/page-bundles/) are used and the images array is empty or undefined, images with file names matching *feature* or *cover*,*thumbnail* are used for image metadata.". 

What we can do is change our card partial and shortcode to use the 1st image in the `images` array if it is not a gif. 

However the other interesting thing that this bit of their docs points to is that there is a videos array that's natively part of the front matter.  We could probably use that instead of webmSrc and mp4Src and use both based on the file type.

Those are somewhat complex changes because our `list.typesense.json` file (which populates our tutorials database) currently assumes there is an `image` or a `webmSrc` and `mp4Src` variable in the front matter.

I'd suggest doing that in 3 parts:
Changing everything for images and then changing everything for videos and if it works, removing gifs as a third step. Would you like to do that work? Absolutely ok if that goes beyond what you want to do here but I want to give you the option :).